packages/eslint-config-rbflabs/README.md

```diff
@@ -0,0 +1,37 @@
+# eslint-config-rbflabs
+
+## Install
+
+`eslint-config-rbflabs` needs to be installed with all of it peerdependencies. In your project run:
+
+```sh
+npx install-peerdeps --dev eslint-config-rbflabs
```

Comment:
Why exactly is this needed? I'm not sure about yarn but npm 7+ install all peer dependencies automatically. Or is this a universal solution for any package manager?

Reply:
Npm will, yarn will not I believe. This also adds the packages to your `devDependencies` so you can make some project specific decisions.